.ci.yaml

```diff
@@ -59,7 +59,9 @@ platform_properties:
           {"name":"flutter_cocoapods","path":"cocoapods"},
           {"name":"old_osx_sdk","path":"osx_sdk"},
           {"name":"builder_mac_engine","path":"builder"},
-          {"name":"openjdk","path":"java"}
+          {"name":"openjdk","path":"java"},
+          {"name":"xcode_runtime_ios-13-0","path":"xcode_runtime_ios-13-0"},
+          {"name":"xcode_runtime_ios-15-0","path":"xcode_runtime_ios-15-0"}
```

Comment:
Something like this
https://flutter.googlesource.com/recipes/+/refs/heads/main/recipe_modules/osx_sdk/api.py#110